docusaurus/docs/Android/01-basics/04-getting-started.mdx

```diff
@@ -168,16 +267,15 @@ ChatClient.instance().connectUser(user, "userToken")  // Replace with a real tok
                 // Handle error
             }
         });
-
 ```
 </TabItem>
 </Tabs>
 
 :::note
-To learn about how to create a token and different user types, see [Tokens & Authentication](https://getstream.io/chat/docs/android/tokens_and_authentication/?language=kotlin).
+To learn more about how to create a token and different user types, see [Tokens & Authentication](https://getstream.io/chat/docs/android/tokens_and_authentication/?language=kotlin).
```

Comment:
It would be helpful to add a link to `Handling User Connection` page. Maybe something like this:
`Make sure to check [Handling User Connection] page if you are looking for more complex scenarios`

Reply:
Added it as a note below saying:

```markdown
:::note
For more information on handling complex scenarios, check out our [Handling User Connection](../02-client/06-guides/04-handling-user-connection.mdx) guide.
:::
```